src/mono/wasm/test-main.js

```diff
@@ -217,6 +218,12 @@ function processQueryArguments(incomingArguments) {
             } else {
                 console.warn("--fetch-random-delay only works on browser")
             }
+        } else if (currentArg.startsWith("--config-src=")) {
+            const arg = currentArg.substring("--config-src=".length);
+            runArgs.configSrc = arg;
+        } else if (currentArg == ("--deep-work-dir")) {
```

Comment:
we could remove the empty condition

Reply:
If we remove that, it will go into the `else` clause and the while loop will `break`.
Unhandled arguments will then be passed as `applicationArgument` , which may cause unintended behavior.